### DISCUSSION TITLE
1주차 스프린트 관련

### DISCUSSION THREAD

**Discussion by jgjgill** [prgrms-fe-devcourse/FEDC4_HONKOK_JunilHwang#14] [General]
1주차 오프라인 스프린트 시 이야기하고 싶은 부분들을 적어둡니다.

그때 그때 생각나는 것들을 자유롭게 적어주세요!

**Comment by jgjgill** [comment]
ESLint로 rushstack의 플러그인을 사용 중입니다.
이와 관련해서 작업하는데 번거로운 규칙도 존재하는 것 같습니다.
작업하면서 불편했던 규칙들에 대해 이야기했으면 하네요..!

**Reply by jgjgill** [reply]
`@rushstack/typedef-var`, `explicit-function-return-type` 이거는 필요 없을 것 같습니다..

**Reply by backward99** [reply]
저도 적극 동의합니다...!

**Comment by jgjgill** [comment]
pr 템플릿에서 이슈 번호가 주석이 기본값이어서 불편하게 느껴지네요..
pr에는 issue가 대부분 붙을 것 같아서 변경하면 더 좋을 것 같습니다..

**Reply by backward99** [reply]
어...저는 이슈에서 branch를 생성해서 그런지 이슈가 자동으로 연결되어져 있더라고요...! 이슈가 연결되어 있지 않을 때도 있을까 해서 주석으로 달아뒀습니다..! 이슈가 연결되어있을 떄는 바로 지워버리면 되니까 주석을 없애버릴까요?

**Comment by backward99** [comment]
공통 세팅?이 끝나면 역할 분담을 어떻게 진행하게 될까요?

**Comment by hyoribogo** [comment]
mutation, query 등 함수명, 변수명, 객체명 등등등 규칙 정하기

**Comment by jgjgill** [comment]
`@tanstack/eslint-plugin-query` 설치 진행하기

**Comment by jgjgill** [comment]
페어 프로그래밍 진행해보기

**Comment by jgjgill** [comment]
폰트, 컬러, 아이콘 정하기

**Comment by jkea1** [comment]
개인적으로 react-query 부분 사용이 아직 미숙해서 오프라인 만났을때 한번 더 뭐 하나 구현해 보고 질문 많이 해야지..

**Comment by jgjgill** [comment]
나중에 한 분이 import 정렬 절대 경로 설정

**Reply by backward99** [reply]
- import 경로 별칭을 어느 범위까지 설정하는게 좋을까요? 각 폴더에 index.ts 파일을 이용해서 내보내기를 진행하면 src바로 밑에 있는 폴더까지 내보낼 수 있는 것 같아서요!
- 경로 별칭은 @로 하는게 나을까요? ~이나 다른거 사용하고 싶은 문자가 있을까요?
-  import 문을 폴더순서에 대해서도 정렬하는게 나을까요?
